controller/workitemtype.go

```diff
@@ -80,10 +82,24 @@ func (c *WorkitemtypeController) Create(ctx *app.CreateWorkitemtypeContext) erro
 			ctx.Payload.Data.Attributes.Name,
 			ctx.Payload.Data.Attributes.Description,
 			ctx.Payload.Data.Attributes.Icon,
-			modelFields)
+			modelFields,
+		)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, err)
+		}
+
+		var category []*uuid.UUID
```

Comment:
The name should reflect the contents:

1. It is an array, hence make the noun plural
2. You store IDs and not the complete categories, so add an `IDs` to the name

You should rename this to `categoryIDs` IMHO